src/avro_util.erl

```diff
@@ -587,6 +654,21 @@ sub_types(T) when ?IS_MAP_TYPE(T) -> [avro_map:get_items_type(T)];
 sub_types(T) when ?IS_ARRAY_TYPE(T) -> [avro_array:get_items_type(T)];
 sub_types(_) -> [].
 
+%% @private
```

Comment:
please remove @private tag.
you may leave the existing ones alone, but we should not add new.